cloudinit/controlplane_init.go

```diff
@@ -24,9 +24,9 @@ const (
     permissions: '0640'
     content: |
       ---
-{{.ClusterConfiguration | Indent 6}}
+{{.ClusterConfiguration}}
```

Comment:
`| Indent 6` is a jinja templating things... removed

Reply:
It was a go template function we wrote in CAPA. I'd recommend bringing it over. Otherwise I don't think it will be valid yaml?

Reply:
Gotcha.
The `Indent` go template func is already there, but it expects `int, string`, while `ClusterConfiguration` ATM is `[]byte`. Fix in flight